app/_src/gateway/upgrade-v2.md

```diff
@@ -127,6 +127,34 @@ The following table outlines various upgrade path scenarios to {{page.kong_versi
 
 {% endif_version %}
 
+{% if_version eq: 3.5.x %}
+
+| **Current version** | **Topology** | **Direct upgrade possible?** | **Upgrade path** |
+| ------------------- | ------------ | ---------------------------- | ---------------- |
+| 2.x–2.7.x | Traditional | No | [Upgrade to 2.8.2.x](/gateway/2.8.x/install-and-run/upgrade-enterprise/) (required for blue/green deployments only), [upgrade to 3.0.x](/gateway/3.0.x/upgrade/), [upgrade to 3.1.x](/gateway/3.1.x/upgrade/#migrate-db), upgrade to 3.2.x, upgrade to 3.3.x, upgrade to 3.4.x, and then upgrade to 3.5.x. |
```

Comment:
@cloudjumpercat , these steps aren't required with LTS > LTS upgrade support. I think we should be describing path as 2.1-2.7 > 2.8 > 3.4 > 3.5. @lena-larionova, wdyt?

Reply:
I think that's a fair update to make. If it's wrong, they'll tell us.

Reply:
@acgoldsmith @lena-larionova I updated the paths, let me know if any are incorrect. For some of them that required upgrading to a specific version (like 2.8 > 3.0.x or 3.1.1.3), I left those because I think there was a reason why they required those incremental upgrades, and I'm not sure if they still apply or not, so I was just playing it safe.